.github/workflows/deploy-all.yml

```diff
@@ -3,8 +3,8 @@ name: Deploy to OpenShift/Camunda
 on:
   workflow_dispatch:
     inputs:
-      buildRef:
-        description: "Build Ref"
+      gitRef:
```

Comment:
I believe that the `default: "main"` does not make much sense anymore.
We can leave it blank and maybe provide a description like `Git Ref (e.g, v1.0.0 or v1.0.0-prerelease.1)`.